modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-api-usage.adoc

```diff
@@ -0,0 +1,41 @@
+[[api-usage]]
+== API Usage
+
+The {pluginname} On-Premises converter provides the ability to convert an HTML document to a PDF file via Restful API.
+
+The API is available on `http://localhost:[port]` (by default the `port` is `8080`).
```

Comment:
URL not visible on the renedered page. Should read `http://localhost:[port]`

Reply:
```suggestion
The API is available on `+http://localhost:[port]+` (by default the `port` is `8080`).
```